specification/indices/add_block/IndicesAddBlockResponse.ts

```diff
@@ -23,11 +23,11 @@ export class Response {
   body: {
     acknowledged: boolean
     shards_acknowledged: boolean
-    indices: IndicesBlockStatus[]
+    indices: AddIndicesBlockStatus[]
   }
 }
 
-export class IndicesBlockStatus {
+export class AddIndicesBlockStatus {
```

Comment:
Thinking about future API extensions: If there is the chance for a `get_index_block_status` endpoint in the future, it would probably make sense to have this type in the `_types` namespace.